packages/helia/src/index.ts

```diff
@@ -91,6 +96,12 @@ export interface HeliaInit<T extends Libp2p = Libp2p> {
    */
   dagWalkers?: DAGWalker[]
 
+  /**
+   * A list of strategies used to fetch blocks when they are not present in
```

Comment:
We should update the comment unless we change the name to something like `BlockStrategies` or `BlockExchangeStrategies`.

```suggestion
   * A list of providers used to fetch blocks when they are not present in
```

Reply:
I intentionally didn't use "providers" here because of the terminology clash with DHT provider records.